dotnet/Services/ProductReviewService.cs

```diff
@@ -331,7 +331,7 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
             AppSettings settings = await GetAppSettings();
             if (settings.RequireApproval)
             {
-                searchQuery = $"{searchQuery} AND approved=true";
```

Comment:
You could probably move this **settings.RequireApproval** below into the if, else conditions since that is where the different conditions have been considered. 
You could take a look at my previous PR description: https://github.com/vtex-apps/reviews-and-ratings/pull/154

Reply:
Ok, i changed this code block to the if statement below, I think this can reduce the complexity, thanks.

Reply:
Oh, with these if statements, the ternary operator is not necessary.